_summaries/ba2016layer_2.md

```diff
@@ -1,19 +1,70 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+title: Summary for Layer Normalization (Ba et al, 2016)
+paper: ba2016layer
+author: jaywhang
+score: 8
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+
+In this paper, the authors propose _Layer Normalization_, an alternative to Batch Normalization (BN) for improving training stability and convergence.
+The core idea is to derive the shift and scale parameters for normalization **within the layer itself**, unlike BN which relies on batch statistics.
+An important distinction to BN is that all hidden units of a layer share the same normalization terms, but may be normalized differently across training examples.
+
+### What issues does it solve?
+
+Layer Normalization solves three major drawbacks of BN:
+1. BN's normalization terms are computed from the minibatch, so it cannot be used when batch size is 1 or small.
+2. BN's normalization terms are updated as an running mean/stddev during training, so there is a mismatch between training and test time.
+3. When applied to a Recurrent Neural Network (RNN), BN requires keeping track of separate batch statistics for each time step. This is problematic when the sequence length is longer at test time, as there would not be a corresponding statistics to normalize with.
+
+In short, Layer Normalization can be applied regardless of the batch size, has identical behavior at training and test time, and can be seamlessly applied to RNNs.
+
+## Technical Description
+
+For a layer $$a$$ with $$H$$ neurons, Layer Normalization computes the mean and standard deviation across the neurons themselves:
+
+$$
+\mu = \frac{1}{H}\sum\limits_{i=1}^{H} a_i, \hspace{3em} \sigma = \sqrt{\frac{1}{H}\sum\limits_{i=1}^{H}(a_i-\mu)^2}
+$$
+
+Note that all neurons ($$a_1, \ldots, a_H$$) share the same scalar parameters $$\mu$$ and $$\sigma$$.
+
+## Invariance Properties
+
+Layer normalization makes different trade-offs compared to the two related techniques, namely Batch Normalization and Weight Normalization.
+The following table summarizes the trade-offs clearly:
+![Table 1: Trade-off comparison to BN and WN.](ba2016layer_2_table1.png)
+
```

Comment:
You can add here about how the normalization reduces learning rate, by showing equation 14 and writing one line on it